test/src/main/java/org/apache/accumulo/test/ComprehensiveIT.java

```diff
@@ -133,6 +151,58 @@ public static void teardown() {
     SharedMiniClusterBase.stopMiniCluster();
   }
 
+  @Test
+  public void testEventualScan() throws Exception {
+    Properties props = new Properties();
+    props.putAll(getClientProps());
+    props.put(ClientProperty.SCAN_SERVER_SELECTOR.getKey(),
+        ConfigurableScanServerSelector.class.getName());
+    props.put(ClientProperty.SCAN_SERVER_SELECTOR_OPTS_PREFIX.getKey() + "profiles",
+        "[{'isDefault':true,'timeToWaitForScanServers' : '10d','maxBusyTimeout':'5m','busyTimeoutMultiplier':4,'attemptPlans':[{\"servers\":\"100%\", \"busyTimeout\":\"3ms\"}]}]");
+
+    try (AccumuloClient client = Accumulo.newClient().from(props).build()) {
+
+      String table = getUniqueNames(1)[0];
+      client.tableOperations().create(table);
+
+      // Start Scan Server
+      getCluster().getClusterControl().start(ServerType.SCAN_SERVER);
+
+      // generate 100 rows of data
+      write(client, table, generateMutations(0, 100, tr -> true));
+
+      // need flush table so that eventual scan can see it.
+      getCluster().getClusterControl().stop(ServerType.SCAN_SERVER);
+      client.tableOperations().flush(table, null, null, true);
+      getCluster().getClusterControl().start(ServerType.SCAN_SERVER);
```

Comment:
Should not need to start and stop the server here.  At this point the tablet has not been accessed via the scan server so the first time the tablet is read on the scan server it will see the files from this flush. 

```suggestion
      client.tableOperations().flush(table, null, null, true);
```